genologics/config.py

```diff
@@ -49,8 +54,7 @@ def load_config(specified_config = None):
             config_file = conf_file[0]
 
         except:
-            warnings.warn("Please make sure you've created or indicated your own Genologics configuration file (i.e: ~/.genologicsrc) as stated in README.md")
```

Comment:
What's going on here?

Reply:
So we used to die on configuration missing. And I thought it made sense, it forced the users to actually have a configuration file, else they couldnt do anything.
But forcing users to have a configuration file in a predefined location is bad practice, while it makes things 'simple' for casual users, it makes like complicated for users with more than one lims, for instance.
This is the reason behind @juliambrosman  PRs to allow multiple config files. 
While there are a couple of things I do not agree with in her PR, since my comments on her PR have been left unanswered for a while, I took it upon myself to implement some of the things she wanted to have in the main repo.